test/zdtm/static/netns_lock.c

```diff
@@ -8,6 +8,7 @@ const char *test_author = "Zeyad Yasser <zeyady98@gmail.com>";
 #include <sched.h>
 
 #define NS_PATH "/var/run/netns/criu-net-lock-test"
+#define SYNCFILE_PATH "/tmp/criu-net-lock-test-sync"
```

Comment:
It would be better to use a relative path instead, for example:
```diff
-#define SYNCFILE_PATH "/tmp/criu-net-lock-test-sync"
+#define SYNCFILE_PATH "zdtm/static/net_lock.sync"
```

Reply:
Thanks, that looks more elegant.